openassessment/templates/openassessmentblock/staff_area/oa_student_info_assessment_detail.html

```diff
@@ -0,0 +1,56 @@
+    <div class="ui-staff ui-toggle-visibility is--collapsed">
+        <h2 class="staff-info__title ui-staff__subtitle ui-toggle-visibility__control">
+            <i class="icon fa fa-caret-right" aria-hidden="true"></i>
+            <span>{% trans translated_title %}</span>
+        </h2>
+        <div class="ui-toggle-visibility__content">
+            {% for assessment in assessments %}
+                {% with assessment_num=forloop.counter %}
+                    {% if len(assessments) > 1 %}
+                        <h4 class="title--sub">
+                            {% blocktrans %}
+                                Assessment {{ assessment_num }}: #TODO: Does this really need to very between "Peer {}:" and "Assessment {}:"? The latter seems cleaner to me in all cases, especially given we already have a translated_title above
+                            {% endblocktrans %}
+                        </h4>
+                    {% endif %}
+                    <table class="staff-info__status__table" summary="{% trans "Assessment" %}">
+                        <thead>
+                        <tr>
+                            <th abbr="Criterion" scope="col">{% trans "Criterion" %}</th>
+                            <th abbr="Selected Option" scope="col">{% trans "Selected Option" %}</th>
+                            <th abbr="Feedback" scope="col">{% trans "Feedback" %}</th>
+                            <th abbr="Points" scope="col">{% trans "Points" %}</th>
+                            <th abbr="Points Possible" scope="col">{% trans "Points Possible" %}</th>
+                        </tr>
+                        </thead>
+
+                        <tbody>
+                        {% for criterion in rubric_criteria %}
+                            {% for part in assessment.parts %}
+                                {% if part.option.criterion.name == criterion.name %}
+                                    <tr>
+                                        <td class="label">{{ criterion.label }}</td>
+                                        <td class="value">{{ part.option.label }}</td>
+                                        <td class="value">{{ part.feedback }}</td>
+                                        <td class="value">{{ part.option.points }}</td>
+                                        <td class="value">{{ criterion.total_value }}</td>
+                                    </tr>
+                                {% endif %}
+                            {% endfor %}
+                        {% endfor %}
+                        </tbody>
+                    </table>
+                    {% if assessment.feedback %}
+                        <h4 class="title--sub">{% trans "Overall Feedback" %}</h4>
+                        <div class="student__answer__display__content">
+                            {{ assessment.feedback|linebreaks }}
```

Comment:
Turns out feedback can exist on these, it just wasn't being displayed before. Do we want to keep hiding it?